### DISCUSSION TITLE
Varied size of control and test

### DISCUSSION THREAD

**Discussion by manojbalaji1** [uber/causalml#309] [Q&A]
I wanted to know does the size difference between control and test matter for meta learner model? Say Test group is 10x of Control.

**Comment by manojbalaji1** [comment]
@paullo0106 wanted to check with you and your team about this

**Comment by t-tte** [comment]
Yes, this matters for meta-learners, and in fact it was one of the motivations for the X-learner. For more details and a comparison between different meta-learners, please refer to the simulation results from p. 5 onwards in [this paper](https://arxiv.org/pdf/1706.03461.pdf).
